src/views/DashboardPage/index.js

```diff
@@ -6,41 +6,99 @@ import Button from "react-bootstrap/Button";
 import { Link, Route } from "react-router-dom";
 import HomePage from "../HomePage";
 import LoginPage from "../LoginPage";
+import DashboardSidebar from "../../components/DashboardSidebar";
+import DashboardCard from "../../components/DashboardCard";
+import styles from "../../components/DashboardSidebar/styles.module.css";
+import { Row, Col, Container } from "react-bootstrap";
+import AWS from "aws-sdk";
 
 const poolData = {
   UserPoolId: process.env.REACT_APP_COGNITO_USER_POOL_ID,
   ClientId: process.env.REACT_APP_COGNITO_CLIENT_ID,
 };
 
+const applicationDeadline = new Date(
+  process.env.REACT_APP_APPLICATION_DEADLINE
+);
+const decisionDate = new Date(process.env.REACT_APP_DECISION_DATE);
+const today = new Date();
+
+const dynamoDbData = {
+  accessKeyId: process.env.REACT_APP_AWS_ACCESS_KEY,
+  secretAccessKey: process.env.REACT_APP_AWS_SECRET_KEY,
+  region: process.env.REACT_APP_AWS_REGION,
+};
+
+const tableName = process.env.REACT_APP_DYNAMO_DB_TABLE;
+
 const userPool = new CognitoUserPool(poolData);
 var cognitoUser = userPool.getCurrentUser();
+var dynamoDB = new AWS.DynamoDB(dynamoDbData);
 var isLoggedIn = false;
 
-window.onload = function () {
-  if (cognitoUser != null) {
-    cognitoUser.getSession(function (err, session) {
-      if (err) {
-        return;
-      }
-      //Set the profile info
-      cognitoUser.getUserAttributes(function (err, result) {
-        if (err) {
-          return;
-        }
-      });
-      isLoggedIn = true;
-    });
-  } else {
-    document.getElementByIdById("display_error").innerHTML = "Not Logged In";
-  }
-};
-
```

Comment:
I'm not super well-versed in React haha, so I'm kinda curious, what was this doing and why were we able to get rid of it?

Reply:
It was using the window.onLoad() function. Dunja wanted to use something from the React component lifecycle, so I just moved it to the constructor (which is below)

Reply:
Yeah they accomplish the same thing, but this was is just more react-y